SimplyE/Colors.xcassets/fieldBorder.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
+  "colors" : [
+    {
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "0x93",
+          "green" : "0x8D",
+          "red" : "0x8D"
+        }
+      },
+      "idiom" : "universal"
+    },
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "dark"
+        }
+      ],
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "0.576",
```

Comment:
do you mind changing these to the #8D8D93 HEX values? they are a lot easier to cross reference.

Reply:
All colors are now using HEX values